qe_tools/parsers/qeinputparser.py

```diff
@@ -396,23 +390,23 @@ def str01_to_bool(s):
         )
         """, re.X | re.M)
 
-    atomic_positions_block_re_ = re.compile(
-        r"""
-        ^ [ \t]* ATOMIC_POSITIONS [ \t]*
-            [{(]? [ \t]* (?P<units>\S+?)? [ \t]* [)}]? [ \t]* $\n
-        (?P<block>
-         (?:
-          ^ [ \t]*
-          (?:
-           \S+ [ \t]+ \S+ [ \t]+ \S+ [ \t]+ \S+
-           (?:[ \t]+ [{(]? [ \t]* [01] [ \t]+ [01] [ \t]+ [01] [ \t]* [)}]?)?
-          )
-          [ \t]* $\n?
-         )+
-        )
-        """, RE_FLAGS)
-    # Define re for atomic positions without force modifications.
+    # atomic_positions_block_re_ = re.compile(
```

Comment:
This was flagged as unused, but I'm not sure if it's supposed to do something or is just old code.

Reply:
Probably leftovers - we might want to remove it altogether in the future